recipe/meta.yaml

```
@@ -1,40 +1,62 @@
 {% set name = "validators" %}
-{% set version = "0.18.2" %}
-{% set sha256 = "37cd9a9213278538ad09b5b9f9134266e7c226ab1fede1d500e29e0a8fbb9ea6" %}
+{% set version = "0.34.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: {{ sha256 }}
+  sha256: 647fe407b45af9a74d245b943b18e6a816acf4926974278f6dd617778e1e781f
 
 build:
-  noarch: python
   number: 0
-  script: {{ PYTHON }} -m pip install . -vv
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  skip: True  # [py<38]
 
 requirements:
   host:
-    - python
     - pip
+    - python
+    - wheel
+    - setuptools
   run:
     - python
-    - six >=1.4.0
-    - decorator >=3.4.0
+  run_constrained:
+    # https://github.com/python-validators/validators/blob/0.34.0/pyproject.toml#L51
+    - eth-hash >=0.7.0
+    - pycryptodome >=3.6.6,<4
+
+# eth-hash is not avaliable in main channel.
+{% set ignore_tests = " --ignore=tests/crypto_addresses/test_eth_address.py" %}
 
 test:
+  source_files:
+    - tests
   imports:
     - validators
+    - validators.crypto_addresses
     - validators.i18n
+  commands:
+    - pip check
+    - python -c "from importlib.metadata import version; assert(version('{{ name }}')=='{{ version }}')"
+    - pytest -v {{ ignore_tests }} tests
+  requires:
+    - pip
+    - pytest
```

```suggestion
    - pytest >=8.3.2
```
https://github.com/python-validators/validators/blob/0.34.0/pyproject.toml#L74